processor/processortest/noop_processor.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019, OpenCensus Authors
```

Comment:
Why is `noopProcessor ` under `processortest`?

Reply:
Because this is mostly used in tests. It is a test util in my opinion and should never be used in production.